packages/form-layout/test/visual/lumo/form-layout-auto-responsive.test.js

```diff
@@ -0,0 +1,100 @@
+import { nextFrame } from '@vaadin/testing-helpers';
+import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
+import { visualDiff } from '@web/test-runner-visual-regression';
+import '../../../theme/lumo/vaadin-form-layout.js';
+import '../../../theme/lumo/vaadin-form-item.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+registerStyles(
+  'vaadin-form-layout',
```

Comment:
This needs to be moved to separate file which needs to be imported before `vaadin-form-layout.js` - otherwise there will be warnings when switching to use Lit version of the component by default.

Reply:
Thanks, removed this block and set the border using inline styles.